### ISSUE TITLE
Use a proxy server

### ISSUE THREAD

**Issue by diegozimet** [ProxymanApp/Proxyman#1572]
Is it possible to configure proxyman to send data through a proxy server? I'm using the iPhone app, and I would like to use a proxy server for cellular data as well, not just Wifi.

Thank you.

**Comment by NghiaTranUIT** [comment]
It's possible for the WiFi. Here is how you setup:
1. Open Proxyman for macOS
2. Get your iPhone iOS 16.3 or later, set the Proxy and install / trust the certificate (https://docs.proxyman.io/debug-devices/ios-device)
3. Make sure you set up properly, so you can see the HTTPS traffic from your iPhone. To test it, just simply open the Safari iOS app -> Visit Google.com -> Back to the macOS app -> Enable SSL this domain -> Verify you can see HTTPS content.
4. To use the Proxy Server, go to Proxyman for macOS -> Tools -> Proxy Setting -> External Proxy
5. Enter your IP/Port for the HTTP and HTPS.
6. Done.

Your iPhone -> Proxyman -> External Proxy -> The Internet.

You can read more at https://docs.proxyman.io/advanced-features/external-proxy

**Comment by doroved** [comment]
> It's possible for the WiFi. Here is how you setup:
> 
> 1. Open Proxyman for macOS
> 2. Get your iPhone iOS 16.3 or later, set the Proxy, and install / trust the certificate (https://docs.proxyman.io/debug-devices/ios-device)
> 3. Make sure you set it up properly, so you can see the HTTPS traffic from your iPhone. To test it, just simply open the Safari iOS app -> Visit Google.com -> Back to the macOS app -> Enable SSL on this domain -> Verify you can see HTTPS content.
> 4. To use the Proxy Server, go to Proxyman for macOS -> Tools -> Proxy Setting -> External Proxy
> 5. Enter your IP/Port for the HTTP and HTTPS.
> 6. Done.
> 
> Your iPhone -> Proxyman -> External Proxy -> The Internet.
> 
> You can read more at https://docs.proxyman.io/advanced-features/external-proxy

It doesn't work. I installed an external proxy, but the traffic is still proxied through my ISP's internet. What could be the problem?

<img width="621" alt="image" src="https://github.com/ProxymanApp/Proxyman/assets/30116541/536be5a3-5eee-4165-8c29-63864a769773">

**Comment by doroved** [comment]
@NghiaTranUIT 

Burp Siute has a proxy setting that allows you to pass all proxied traffic through an external proxy.
I take it your program does not have this?
<img width="932" alt="image" src="https://github.com/ProxymanApp/Proxyman/assets/30116541/8319aa6e-8f50-4b02-984c-15088bb253be">

**Comment by NghiaTranUIT** [comment]
@doroved It's the External proxying Feature in the Tool menu -> Proxy Setting.

- Destination host: Include List
- We don't support Domain or Domain host in the authentication type.

<img width="718" alt="Screenshot 2023-05-25 at 06 35 44" src="https://github.com/ProxymanApp/Proxyman/assets/5878421/6d667143-b0b2-42dc-b061-b724a88b63a6">

**Comment by doroved** [comment]
> 

I enter ip,port,auth credentials external proxy like this, but it does not work, the traffic does not go through the external proxy
![image](https://github.com/ProxymanApp/Proxyman/assets/30116541/4a36c21f-1fce-4d8f-bcb9-5f9c53405db8)

**Comment by NghiaTranUIT** [comment]
What is your URL? Is it localhost request?

**Comment by NghiaTranUIT** [comment]
If it's localhost url, please uncheck the Always bypass the localhost in the end of the box 👍

**Comment by doroved** [comment]
> What is your URL? Is it localhost request?

No, for example I go to the site to check my ip and it does not change [ifconfig.co](https://ifconfig.co).
By the way, in Charles, my external proxy does not work either.
I closed all applications proxy traffic and manually set proxy in wi-fi, also does not work, even no error, traffic does not go through external proxy and goes through my provider's internet.
I can not understand what the problem may be.

**Comment by NghiaTranUIT** [comment]
Can you verify that you use HTTPS on the External Proxy too?

From what I see, you only enable HTTP.

![CleanShot 2023-05-25 at 14 30 04@2x](https://github.com/ProxymanApp/Proxyman/assets/5878421/1be293dd-4780-45f2-86d6-592048661768)

**Comment by doroved** [comment]
> Can you verify that you use HTTPS on the External Proxy too?
> 
> From what I see, you only enable HTTP.

**I added the same proxy for HTTPS and everything worked, thanks)**
It confused me that for HTTPS the default port is 443, I have these proxies, they work through the domain instead of ip, so I specified my HTTP proxies in Proxyman's HTTP settings

**Comment by NghiaTranUIT** [comment]
Ah, I see. I will change the placeholder of the HTTPS Box to make it clear 👍
